.circleci/config.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - run: git clone --depth 1 https://github.com/sunpy/sunpy.org.git
       - run: cd sunpy.org && git submodule update --init && cd ..
       - run: pip install -r sunpy.org/requirements.txt
-      - run: cd sunpy.org && sphinx-build -W --color -b html -d _build/.doctrees . _build/html
+      - run: cd sunpy.org && sphinx-build -j auto -W --color -b html -d _build/.doctrees . _build/html
```

Comment:
I would build one or more of these in serial so we are testing both

Reply:
Done.